samples/ios/Continote/Continote/MyNotesTableViewDataSource.swift

```diff
@@ -34,8 +35,7 @@ class MyNotesTableViewDataSource: FUITableViewDataSource {
   override func tableView(_ tableView: UITableView,
                           commit editingStyle: UITableViewCellEditingStyle,
                           forRowAt indexPath: IndexPath) {
-    switch editingStyle {
-    case .delete:
+    if editingStyle == .delete {
```

Comment:
Sorry should have mentioned before but didn't stick out to me - one thing I'm fond of doing if the entire function is dependent on a condition is throw it in a guard statement so you don't have to nest the entire function one block deeper:
`guard editingStyle == .delete else { return }`
at the top will remove one more set of braces!